internal/types/client_utils.go

```diff
@@ -37,6 +37,74 @@ func ToStateAuthenticationPolicyContract(r *client.AuthenticationPolicyContract)
 	return caSliceOfObj, eaSliceOfObj
 }
 
+func AttrValueToBoolPointer(val attr.Value) *bool {
+	boolPointerVal := ConvertToPrimitive(val).(bool)
+	return &boolPointerVal
+}
+
+func ToStateRedirectValidation(r *client.RedirectValidationSettings, diags diag.Diagnostics) (basetypes.ObjectValue, basetypes.ObjectValue) {
```

Comment:
If this only applies to this new resource I would move it to the resource Go file

Reply:
Fixed in the latest commit